app/javascript/styles/components.scss

```diff
@@ -1616,7 +1616,7 @@
   cursor: pointer;
   flex: 0 0 auto;
   font-size: 16px;
-  padding: 0 5px 0 0;
+  padding-right: 15px;
```

Comment:
- Using `padding` is required to overwrite user agent stylesheet.

`padding: 0 5px 0 0;` and `padding: 0 15px 0 0;` for width under 1024px ?
I can't think of any smart ways.. :thinking:

Reply:
or, use `:last-child` instead of media queries?

```
padding: 0 5px 0 0;

&:last-child {
  padding: 0 15px 0 0;
}
```

Reply:
Since there are discussions on columns width, I think using last-child is suitable. Thanks!

Reply:
Thank you for nice suggestion!